# This msg file specifies the arm type
# two potential values (1) "Left" (2) "Right"

string armType